.github/workflows/build_and_test_rx_bloc_packages.yaml

```diff
@@ -50,7 +51,7 @@ jobs:
           fi
       - name: Remove unnecessary files
         working-directory: ${{ matrix.folder }}
-        run: flutter pub run clean_coverage clean --exclusions '**/.g.dart','**repository.dart','**rxb.g.dart' coverage/lcov.info
+        run: flutter pub run clean_coverage clean --exclusions '**/.g.dart','**/di/**','**repository.dart','**/models/**','**/routes.dart','**data_source.dart','**_model.g.dart','**_models.g.dart''**_model.dart','**_model_extensions.dart','**_models.dart','**_models_extensions.dart','**_config.dart','**_constants.dart','**_translations.dart','lib/assets.dart','lib/keys.dart','lib/base/theme/**','lib/base/utils/**','lib/keys.dart','lib/l10n/**' coverage/lcov.info
```

Comment:
@StanevPrime I've removed your comment thinking that I would remove mine by mistake. You've asked if we can include this into the rx_bloc_cli package, and I can say to that yes, we can. Will add it in a separate PR along with some other smaller changes.

Reply:
Ok, please proceed. The end goal is to have code coverage report only for the classes that can be tested.